src/BloomExe/Collection/CollectionSettings.cs

```diff
@@ -495,6 +495,19 @@ public void Load()
 			{
 				DoOneTimeCheck();
 			}
+
```

Comment:
I wonder if this is unnecessary work and complexity. What would be the downside to simply calling SaveSettingsCollectionStylesCss() after each Load()? An upside would be that know we are always working with what the latest code thinks we need; the next time a bug like this occurs, well, it won't be a bug because it won't happen.

@StephenMcConnel